pkg/object/object.go

```diff
@@ -50,16 +71,129 @@ type OpenCompose struct {
 	Volumes  []Volume
 }
 
+func (s *Service) EmptyDirVolumeExists(name string) bool {
+
+	for _, emptyDirVolume := range s.EmptyDirVolumes {
+		if name == emptyDirVolume.Name {
+			return true
+		}
+	}
+	return false
+}
+
+func (o *OpenCompose) VolumeExists(name string) bool {
+	for _, volume := range o.Volumes {
+		if name == volume.Name {
+			return true
+		}
+	}
+	return false
+}
+
+// Documentation about the valid identifiers can be found at
+// https://github.com/kubernetes/community/blob/master/contributors/design-proposals/identifiers.md
+func isValidName(name string) error {
```

Comment:
>this is called `isValidName` but other validation function are called `validate<something>` ( `validateVolumeMode`
`validatePath`). Wouldn't be better to use same naming scheme for all function validation fields?

done